### PR TITLE
Update FAQ - Dateiverwaltung

### PR DESCRIPTION
**Pull request by fkaminski**
Frage: Kann man die Suche in der Dateiverwaltung ausblenden?

### CODE REVIEW

File: docs/manual/faq/_index.de.md [hunk 1]
```diff
@@ -123,6 +123,15 @@ gekennzeichnet ist.
 Stelle außerdem sicher, dass sich keine veraltete `.htaccess` Datei im Order `/web` oder einem übergeordneten Ordner deiner Installation befindet.
 {{% /expand %}}
 
+{{% expand "Kann man die Suche in der Dateiverwaltung ausblenden?" %}}
+Über einen DCA Eintrag:
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
Ja. Über einen DCA-Eintrag:
```

File: docs/manual/faq/_index.de.md [hunk 2]
```diff
@@ -123,6 +123,15 @@ gekennzeichnet ist.
...
+
+```yml
```
Comment (netzarbeiter) [inline docs/manual/faq/_index.de.md]:
```suggestion
```php
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko

**Comment by fritzmg** [comment]
I am not sure this warrants an FAQ entry? There are thousands of things you can adjust in Contao's DCA - we won't possibly list all of them in the FAQ?

**Comment by Toflar** [comment]
It's still better to keep those snippets in the official documentation rather than keeping them in your personal one. But I agree, we should probably have a `Snippets` section for that?

**Comment by fritzmg** [comment]
Agreed, may be a "DCA adjustments" [guide](https://docs.contao.org/manual/en/guides/) where we can list them in the same manner (but does not have to be phrased like a question then 😉). But for now I suppose we can keep it in the FAQ.

**Comment by fkaminski** [comment]
> Agreed, may be a "DCA adjustments" guide where we can list them in the same manner

Maybe we could start a community call for these collections (BestOf/Awesome-List)?

**Comment by fritzmg** [comment]
> Maybe we could start a community call for these collections (BestOf/Awesome-List)?

I don't think we need a community call - the page just has to be created and then over time filled up by the community :)

**Comment by fkaminski** [comment]
Always looking for appropriate solutions. A consistent place in the documentation would be ideal. I assume that you already have a large number of examples for starting :-) ...

**Comment by fritzmg** [comment]
> I assume that you already have a large number of examples for starting

No, not really ;). This needs to be built over time - i.e. whenever something comes up.

**Review by netzarbeiter — APPROVED** [review]

**Comment by netzarbeiter** [comment]
https://github.com/contao/docs/commit/8989db60eace0ee1ee28c655ba33294b92faed9f

**Pull request merged**